gadget/update_test.go

```diff
@@ -215,6 +215,9 @@ func (u *updateTestSuite) TestCanUpdateSize(c *C) {
 }
 
 func (u *updateTestSuite) TestCanUpdateOffsetWrite(c *C) {
+	// We do not care anymore about changes in offset-write, so nothing
+	// errors here.
+	// TODO Should we just remove the test now?
```

Comment:
we might keep a subset of realistic happy cases and leave also a comment here at the beginning

Reply:
I've reduced the number of tests now